contents/customers/ycombinator.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How Y Combinator uses PostHog experiments to boost engagement by 40%
+customer: Y Combinator
+logo: ../images/customers/ycombinator/yc_logo_b.png
+featuredImage: ../images/customers/ycombinator/featured.svg
+featuredCustomer: true
+industries:
+    - SaaS, Education
+users:
+    - Leadership
+    - Engineering
+    - Product
+toolsUsed:
+    - Experiments
+    - Autocapture
+    - PostHog Cloud
+    - Insights
+---
+
+Y Combinator is the world’s top startup accelerator, helping to fund, train and nurture the next generation of innovative businesses. Since 2005, thousands of successful companies have emerged from Y Combinator, including Dropbox, Airbnb, PagerDuty, Reddit, Amplitude... and PostHog. 
+
+"Many people, including us, find Y Combinator to be the the most productive period of their lives,” said PostHog CEO James Hawkins in [a blogpost he wrote about his Y Combinator experience](/blog/moving-to-sf). Now, PostHog is used by Y Combinator to improve many of the products and processes PostHog benefitted from! 
+
+“We use PostHog for [Startup School](https://www.startupschool.org/), our public [YC Startup Library](https://www.ycombinator.com/library), and for [Co-Founder Matching](https://www.ycombinator.com/cofounder-matching),” said Cat Li, Product and Engineering Lead for Y Combinator Startup School. “PostHog is especially useful for Co-Founder Matching, which is essentially a dating app for co-founders.”
+
+### How Y Combinator gathers 30% more data with PostHog than Google Analytics
+Y Combinator empowers each team to choose its own processes, tools and OKRs. Some teams use tools like Segment or Amplitude — but for the flagship Startup School project, Cat's team decided PostHog was the best tool for the job. 
+
+“Startup School is hugely popular, so we thought a lot about what tools to use,” said Cat. “Many platforms we looked at, including Google Analytics, dropped 30% of user data due to adblockers, or third-party cookies. PostHog became the obvious choice because it didn't have those issues.”
+
+PostHog also offered the benefit of autocapture, which enabled the team to get started quickly without wasting weeks defining tracking plans. 
+
+“Tools we looked at either weren't robust enough, or required a huge amount of work to instrument,” said Cat. “But with PostHog, we could autocapture most events using [the JS snippet](/docs/integrate/client/snippet-installation) and still [configure custom events](tutorials/event-tracking-guide) when we needed to. It was the perfect balance.”
+
+### How Y Combinator boosted Co-founder Matching engagement by 40% 
+Since getting started with PostHog, Y Combinator has gone beyond analytics and begin using other PostHog features, such as [experiments](/product/experimentation) and [session recordings](/product/session-recording), to explore data from multiple angles at once.
```

Comment:
```suggestion
Since starting with PostHog, Y Combinator has gone beyond analytics and begun using more PostHog features, such as [experiments](/product/experimentation) and [session recordings](/product/session-recording), to explore data from multiple angles at once.
```